lib/Runtime/Library/TypedArray.cpp

```diff
@@ -2815,28 +2562,41 @@ namespace Js
         return Js::JavascriptNumber::ToVarNoCheck(currentRes, scriptContext);
     }
 
+    // static 
+    TypedArrayBase * TypedArrayBase::ValidateTypedArray(Arguments &args, ScriptContext *scriptContext, LPCWSTR apiName)
```

Comment:
I would like to leave this as is, sometime when we read spec and  we want to search that function and you find that quickly.

---
In reply to: [80846077](https://github.com/Microsoft/ChakraCore/pull/1629#discussion_r80846077) [](ancestors = 80846077)